.github/workflows/build-release.yml

```diff
@@ -12,22 +12,41 @@ jobs:
         uses: actions/setup-java@v1
         with:
           java-version: 8
+      - name: Cache Iris
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/loom-cache
+            ~/.gradle/wrapper
+            ~/.m2/repository
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
```

Comment:
I assume it's intended that gradle.properties isn't included here?

Reply:
Yes, although it doesn't really matter much tbh. Gradle is pretty smart with its cachings, and custom keys per build.gradle are probably not even needed, but just in case. (This is also the shown example in the cache docs)